taworks/taform/templates/taform/course_list.html

```diff
@@ -1,10 +1,14 @@
+{% load static %}
+<link rel="stylesheet" type="text/css" href="{% static 'taform/css/style.css' %}" />
+
+<form>
+    <button class="button" formaction="{% url 'home' %}">Home</button>
+</form>
 <div>
     <h1>Upload Courses</h1>
 </div>
-
-<p>Please ensure the file you upload is in CSV format. This file must contain the course ID, name, instructor and email information. Click <a href="CourseTemplate.csv" download="CourseTemplate.csv">here</a> for the CSV template. Please note this will remove existing data.
-
-
+<div id="font">{{error}}</div>
+<p>Please ensure the file you upload is in CSV format. This file must contain the course ID, name, instructor and email information. Click <a href="../../static/taform/course_template.csv" download="course_template.csv">here</a> for the CSV template. Please note this will remove existing data.
```

Comment:
ending p tag?
also i think with p tag you can just press enter and have this over multiple lines right?